Dockerfile

```diff
@@ -1,6 +1,8 @@
-FROM python:3.9-slim-bullseye
+#https://stackoverflow.com/questions/77364550/attributeerror-module-pkgutil-has-no-attribute-impimporter-did-you-mean
+#https://github.com/pyproj4/pyproj/issues/1321
+FROM python:3.8-slim-bullseye
```

Comment:
Could you add a TODO + jira ticket here to upgrade this from python 3.8? It doesn't have to be a near term goal, but 3.8 will be EOL soon and I think we should track this